designer/server/src/main/scala/pl/touk/nussknacker/ui/db/timeseries/questdb/QuestDbFEStatisticsRepository.scala

```diff
@@ -237,7 +244,8 @@ object QuestDbFEStatisticsRepository extends LazyLogging {
     if (nuDir.exists) {
       Try(nuDir.delete())
     }
-    createDirAndConfigureLogging()
+    createRootDirIfNotExists(nuDir)
```

Comment:
we have a similar code in the `createCairoEngine` method. Cannot we use this method here?